xrootd/xrdio/parse.go

```diff
@@ -18,7 +18,7 @@ func Parse(name string) (URL, error) {
 		return URL{}, errors.WithStack(err)
 	}
 
-	host := urn.Hostname()
+	host := urn.Host
```

Comment:
right... I went with `Host` instead of `Hostname()` because of IPv6:
`Hostname()` would return `1234:1234:1234:1234` as an IPv6 address instead of `[1234:1234:1234:1234]` so down the line something would break.

(I noticed it when testing against @vkuznet CMS xrootd server that would redirect to an IPv6 server. I can't reproduce now, though...)

I'll switch back to `Hostname()`.